guidelines/speaker-coordination.rst

```diff
@@ -0,0 +1,71 @@
+This document provides set of guidelines for coordinating Speakers for Workshop and Talks.
+
+
+Email guideline
+--------------
+
+- When contacting the Speaker or the participants, the Subject should be "[PYCON-INDIA-{enter year}][WORKSHOP]{reason of the email}".
+
+Todo before contacting the Speakers
+--------------------------
+
+- Call for volunteers, at least 4 volunteers per audi should be sufficient. This excludes the Lead Speaker Coordinator as (s)he will have to run between different audi.
+- Gather contact details of all the volunteers.
+- Do ask the volunteers if there is any specific Workshop/Talk they would like to attend, so that there won't be any missing volunteers at the last minute.
+
+Three weeks before the conference
+----------------------
+
+- Divide the volunteers into different audis and make a list of all speakers in the respective audis.
+- Inform the volunteers to contact the respective Workshop Speakers requesting if there is any addition to the requisite already mentioned in the Proposal.
+- The additional requisite can contain steps to install packages xyz, etc.
+- Request the Workshop Speakers for contact details, so that the workshop participants can contact them if there are any doubts.
+
+Two week before conf
+----------------
+
+- Volunteers can go ahead and introduce the speaker to it's participants with the list of all requisites mentioned by the speaker.
+
+One week before conf
+----------------
+
+- The coordinator should make sure that the volunteers get to attend talks that they prefer and adjust the work force accordingly so that there are enough volunteers in an audi.
+- The speaker coordinator should contact the Volunteer coordinator and get the feedback form, if it is in paper.
+- Make Time card of 15 Minutes, 10 Minutes, 5 Minutes & Times Up.
+
+One day before workshop
+--------------
+
+- Make sure Projectors are set up.
+- Make sure there are at least three wireless mics for questionnaire and two portable mics for the speaker(s).
+
+Note for future coordinators
+------------
+
+- Contact all the speakers for the day and run them through the drill.
+- First day of Talk, do note that DgpLUG team will have a meeting around 11:30 am. So make sure that there are enough volunteers in audis.
```

Comment:
Please remove the reference to dgplug. You can word it as "Volunteers may step out for other sessions/meetup discussion".